apple/OmnivoreKit/Sources/Views/FeedItem/HomeFeedCardView.swift

```diff
@@ -77,7 +77,7 @@ public struct FeedCard: View {
         .padding(.top, 8)
       }
     }
-    .padding(.top, 10)
```

Comment:
@satindar from what I can tell, SwiftUI is now adding the spacing before the item also, so we don't need this padding. Either that or its padding from the cell edge insets